src/napari_workflows/_workflow.py

```diff
@@ -120,6 +120,19 @@ def roots(self):
                             origins.append(source)
         return origins
 
+    def get_root_functions(self):
```

Comment:
I'm thinking of changing that name. Would "root_functions" also work? I'm suggesting that, because "roots" also doesn't have a "get_" prefix...

Reply:
That sounds fine to me!

Reply:
Ok cool. Then please change it and add it to the tests. I'll happily merge it then. No hurry though. Enjoy your Sunday! 🌞